### ISSUE TITLE
Not working with new october firmwares

### ISSUE THREAD

**Issue by cryptage21** [robinelfrink/gs1200-exporter#7]
Hi,

In new firmwares (GS1200-5/5HPv2/8/8HPv2), Zyxel added an encryption algorithm for password authentication.
It seems it's not working anymore.

Are you able to solve this problem ? 

Regards

**Comment by cryptage21** [comment]
Thanks.

Tested on GS1200-5 this morning : it's working.
For information, is we used GS1200-Exporter with previous firmware the program panics : 

`Starting gs1200-exporter.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0x938945]

goroutine 20 [running]:
main.(*Exporter).Collect(0xc0002ce840, 0xc0002d12c0)
	/home/cryptage/gs1200-exporter/main.go:141 +0x1da5
github.com/prometheus/client_golang/prometheus.(*Registry).Gather.func1()
	/home/cryptage/go/pkg/mod/github.com/prometheus/client_golang@v1.11.0/prometheus/registry.go:446 +0x1a2
created by github.com/prometheus/client_golang/prometheus.(*Registry).Gather
	/home/cryptage/go/pkg/mod/github.com/prometheus/client_golang@v1.11.0/prometheus/registry.go:538 +0xe8e
`
